interbase/interbase.php

```diff
@@ -656,15 +655,11 @@ function ibase_server_info ($service_handle, $action) {}
 /**
  * Wait for an event to be posted by the database
  * @link https://php.net/manual/en/function.ibase-wait-event.php
- * @param string $event_name1 <p>
- * The event name.
- * </p>
- * @param string $event_name2 [optional] <p>
- * </p>
- * @param string $_ [optional]
+ * @param string ...$event_names <p>
+ * The event names.
  * @return string the name of the event that was posted.
  */
-function ibase_wait_event ($event_name1, $event_name2 = null, $_ = null) {}
+function ibase_wait_event (...$event_names) {}
```

Comment:
I can't find any proofs of this. Have you checked this?

Reply:
Yes, but it's not 100% correct in both cases. :-/

https://github.com/php/pecl-database-interbase/blob/2044c06d94126115b38de5487cc2119a45aad6bb/ibase_events.c#L125

```
ibase_wait_event ( string ...$event_names) : string
ibase_wait_event ( resource $link_identifier, string ...$event_names) : string
```